### DISCUSSION TITLE
Replace in text

### DISCUSSION THREAD

**Discussion by marc0777** [outline/outline#5316] [Ideas]
Since #5061 has been merged I had to replace all the `$` symbols in my old Markdown files I'm importing into Outline to `$$`. This made me realize that there is no replace function in the editor. Even Chrome extensions such as [this one](https://chrome.google.com/webstore/detail/find-replace-for-text-edi/jajhdmnpiocpbpnlpejbgmpijgmoknnl) don't work. 

In my opinion a good find and replace is fundamental in any editor, and could be even better if it supported RegEx.

Could such a feature be implemented in future version?

**Comment by tommoor** [comment]
Agreed, tracking the issue here https://github.com/outline/outline/issues/4520

**Comment by marc0777** [comment]
Great, thanks! Tried searching between the issues and somehow I missed that.
